### DISCUSSION TITLE
cant resolve node_modules\jpeg-exif\lib

### DISCUSSION THREAD

**Discussion by pablolevipiltw** [diegomura/react-pdf#2563] [General]
hi, After the last update of react-pdf/renderer I have not been able to deploy prod due to problems with the jpeg-exif library. 
![problem](https://github.com/diegomura/react-pdf/assets/119873694/e3256585-6d88-477e-9d42-a68d53b259e7)

**Comment by Yuri-Joel** [comment]
eu também estou com este problema, ajudem me

**Comment by DevDaniloFerrari** [comment]
Se você estiver usando yarn altere suas dependências dessa forma:
```
"resolutions": {
    "@react-pdf/image": "2.2.3",
    "@react-pdf/pdfkit": "3.0.4",
    "@react-pdf/layout": "3.5.0"
  },
```

Se estiver usando npm:

```
"overrides": {
    "@react-pdf/image": "2.2.3",
    "@react-pdf/pdfkit": "3.0.4",
    "@react-pdf/layout": "3.5.0"
  },
```

**Comment by diegomura** [comment]
Fixed in latest versions
